### PR TITLE
feat: new device fields

### PR DESCRIPTION
**Pull request by bruno-garcia**
Displays new `Device` fields sent by the Unity SDK:

image

### CODE REVIEW

**Review by markstory — APPROVED** [review]
Looks good. Should we add a new percy snapshot for the new device data and graphics card metadata that was recently added?

**Comment by benvinegar** [comment]
> Looks good. Should we add a new percy snapshot for the new device data and graphics card metadata that was recently added?

If you want to make sure that the metadata is rendered appropriately – because there are some icons or something that we want to keep visual tabs on – then yes.

But we shouldn't use Percy as a sort of, additional integration test (e.g. to verify stuff was rendered). You can do that in an acceptance test that does some simple assertions on what was rendered.

**Comment by markstory** [comment]
@benvinegar I think this matches that grade to me. There are unique icons and UI state that will only show for unity based events.

**Review by jan-auer — APPROVED** [review]
Generally looks good, two comments below. Afterwards, we should also add the new device context fields to sentry-docs.

File: .gitignore [hunk 1]
```diff
@@ -40,5 +40,4 @@ Gemfile.lock
 coverage.xml
 junit.xml
 *.codestyle.xml
-.pytest_cache/
```
[the remark this answers is not in the document]
Comment (bruno-garcia) [reply .gitignore]: there were two entries

File: src/sentry/static/sentry/app/components/events/contexts/device.jsx [hunk 2]
```diff
@@ -71,6 +73,7 @@ class DeviceContextType extends React.Component {
       external_free_storage,
       boot_time,
       timezone,
+      device_type,
```
[the remark this answers is not in the document]
Comment (bruno-garcia) [reply src/sentry/static/sentry/app/components/events/contexts/device.jsx]: This holds: Handheld, Desktop, Console

[the remark this answers is not in the document]
Comment (jan-auer) [reply src/sentry/static/sentry/app/components/events/contexts/device.jsx]: I still think this should probably go in family, but doesn't hurt for now.

**Pull request merged**